exercises/pangram/description.md

```diff
@@ -2,5 +2,6 @@ Determine if a sentence is a pangram. A pangram (Greek: παν γράμμα, pan
 "every letter") is a sentence using every letter of the alphabet at least once.
 The best known English pangram is "The quick brown fox jumps over the lazy dog."
 
-The alphabet used is ASCII, and case insensitive, from 'a' to 'z'
-inclusively.
+The alphabet used consists of ASCII letters `a` to `z`, inclusive, and is case
+insensitive. Per [#741](https://github.com/exercism/problem-specifications/pull/741),
```

Comment:
That would make more sense to me. Does this change also mean we need to remove tests containing anything other than the letters `a`-`z`? What about spaces?